scanapi/tree/testing_node.py

```diff
@@ -26,11 +26,11 @@ def __getitem__(self, item):
 
     @property
     def name(self):
-        return self["name"]
+        return self[NAME_KEY]
```

Comment:
This is out of scope, but I noticed it was using a different standard from the other files.